data/campaigns/Eastern_Invasion/scenarios/01_Eastern_Invasion.cfg

```diff
@@ -0,0 +1,944 @@
+#textdomain wesnoth-ei
+
+#
+# This is intended to be a difficult mission, moreso than the next few.
+# I think high difficulty is necessary to establish the undead (and particularly Mal-Ravanal) as a credible threat.
+#
+# At first, this scenario is pretty easy. The enemies have low gold and low income.
+# But once Mal-Ravanal shows up, things get dangerous:
+#    1) this (hopefully) makes the player fear Mal-Ravanal right off the bat
+#    2) this establishes the undead invasion as a credible threat.
+#
+# In my expeience, the best strategy is to push forward and gather XP in the first half,
+# then give ground after Mal-Ravanal shows up, during the second night.
+# You need to have enough units/hp to push forward during the third day.
+#
+
+[scenario]
+    id=01_Eastern_Invasion
+    name= _ "Eastern Invasion"
+    map_file=01_Eastern_Invasion.map
+    next_scenario=02_The_Escape_Tunnel
+    turns=11
+    victory_when_enemies_defeated=no
+
+    {DEFAULT_SCHEDULE_DUSK}
+
+    {SETUP_CAMPFIRES}
+
+    {INTRO_AND_SCENARIO_MUSIC main_menu.ogg loyalists.ogg}
+    {LET_INTRO_MUSIC_FINISH}
+    [story]
+        [part]
+            title= _ "Warning!"
```

Comment:
...coming back to this for a moment, I wonder if the double "Warning!" is excessive? Maybe change one of the two to just "Content Note:" or something?

Reply:
...or maybe move it to the campaign description instead? I think just a simple "Content Note: contains mild blood and gore" in the `description=` key in `_main.cfg` would probably preferable, due to concision...